### PR TITLE
OPSEXP-1833 Deploy on nexus 

### PR DESCRIPTION
**Pull request by slohe1**
OPSEXP-1833

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -12,7 +12,9 @@ jobs:
         uses: actions/checkout@v3
       - name: Set imagemagick version
         id: imagemagick 
-        run: echo "version=$(cat ./.github/actions/imagemagick-version)" >> $GITHUB_OUTPUT
+        run: |
+          echo "version=$(cat ./.github/actions/imagemagick-version)" >> $GITHUB_OUTPUT
+          echo "release=$(cat ./.github/actions/release-version)" >> $GITHUB_OUTPUT
```
Comment (alxgomz) [inline .github/workflows/build.yml:17]: I did not realize we had such generic variable names.
Please use something more precise like `imagemagick_version` & `imagemagick_release` that is also more consistent with the filenames

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -50,7 +52,9 @@ jobs:
...
```
Comment (alxgomz) [inline .github/workflows/build.yml:57]: same as above

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -88,7 +92,9 @@ jobs:
...
```
Comment (alxgomz) [inline .github/workflows/build.yml:97]: same as above

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -126,7 +132,9 @@ jobs:
...
```
Comment (alxgomz) [inline .github/workflows/build.yml:137]: same as above

File: .github/workflows/build.yml [hunk 5]
```diff
@@ -164,7 +172,9 @@ jobs:
...
```
Comment (alxgomz) [inline .github/workflows/build.yml:177]: same as above

**Review by alxgomz — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml [hunk 6]
```diff
@@ -12,7 +12,9 @@ jobs:
         uses: actions/checkout@v3
       - name: Set imagemagick version
```
Comment (gionn) [inline .github/workflows/build.yml:13]:
```suggestion
      - name: Set build variables
```

or something similar

File: .github/workflows/build.yml [hunk 7]
```diff
@@ -32,7 +34,7 @@ jobs:
           group-id: org.imagemagick
           artifact-id: imagemagick-distribution
           repository-url: https://nexus.alfresco.com/nexus/content/repositories/thirdparty/
-          version: ${{ steps.build.outputs.built-version }}-ci
+          version: ${{ steps.build.outputs.built-version }}-${{ steps.imagemagick.outputs.imagemagick_release }}-ci
```
Comment (gionn) [inline .github/workflows/build.yml:37]: I think it's better to have -ci as a prefix to always have it in the same place, swap them:

```
${{ steps.build.outputs.built-version }}-ci-${{ steps.imagemagick.outputs.imagemagick_release }}
```

**Review by alxgomz — APPROVED** [review]

**Pull request merged**